package.json

```diff
@@ -4,14 +4,15 @@
   "description": "Library for data processing in pure JavaScript",
   "main": "dist/index.js",
   "dependencies": {
+    "@tensorflow/tfjs": "^2.7.0",
```

Comment:
We should only import tfjs-layers to ensure the package size

Reply:
I was not aware of that. I will change that.